<ui version="4.0" >
 <class>KWin::MouseMarkEffectConfigForm</class>
 <widget class="QWidget" name="KWin::MouseMarkEffectConfigForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>226</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;Width:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>spinWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinWidth" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix" >
         <string> px</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>&amp;Color:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>comboColors</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KColorCombo" name="comboColors" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>Custom...</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Custom...</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KWin::GlobalShortcutsEditor" native="1" name="editor" />
   </item>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Draw with the mouse by holding Shift+Meta keys and moving the mouse.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KWin::GlobalShortcutsEditor</class>
   <extends>QWidget</extends>
   <header location="global" >kwineffects.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
